File: Section_Java/Rectangle/Rectangle.java
public class Rectangle {
    private double longueur;
    private double largeur;
    public Rectangle(double L, double l){
        if(l<0){l=l*(-1);}
        this.largeur=l;
        if(L<0){L=L*(-1);}
        this.longueur=L;
    }
    public double getlongueur(){
        return longueur;
    }
    public double getlargeu(){
        return largeur;
    }
    public void setlongueur(double L){
        if(L<0){L=L*(-1);}
        this.longueur=L;
    }
    public void setlargeur(double l){
        if(l<0){l=l*(-1);}
        this.largeur=l;
    }
    public double perimetre(){
        return 2*(this.largeur+this.longueur);
    }
    public double surface(){
        return this.largeur*this.longueur;
    }
    public void affichage(){
        System.out.println(perimetre());
        System.out.println(surface());
    }


}

File: Heritage/Rectangle.cpp
#include <iostream>
using namespace std;

class Rectangle
{
    private:
    float longueur;
    float largeur;

public:
    Rectangle(float, float);
    float aire();
    float
     perimetre();
};

Rectangle::Rectangle(float L, float l)
{
    if (L<0)
    {
        L=L*(-1);
        longueur=L;
    }
    if (l<0)
    {
        l=l*(-1);
        largeur=l;
    }
}
float Rectangle::aire(){
    return longueur*largeur;
}
float Rectangle::perimetre(){
    return 2*(longueur*largeur);
}
class Cercle: private Rectangle{
    public:
    Cercle(float a);
    void changer_cote(float a);
};

